multiload/linux-proc.h

```diff
@@ -8,7 +8,7 @@ G_GNUC_INTERNAL void GetDiskLoad (int Maximum, int data [3], LoadGraph *g);
 #if 0
 G_GNUC_INTERNAL void GetPage (int Maximum, int data [3], LoadGraph *g);
 #endif /* 0 */
-G_GNUC_INTERNAL void GetMemory (int Maximum, int data [4], LoadGraph *g);
+G_GNUC_INTERNAL void GetMemory (int Maximum, int data [5], LoadGraph *g);
```

Comment:
Why do you set 5 if you only use 4 data sources?

Reply:
We have to pass the zero instead of buffers, otherwise the colors "slide" - the color of "cache" becomes the color of "free". We probably don't want to mess up the user-defined colors; on the other hand, maybe better mess up them instead of messing up the code in this way. See the discussion about by last commit here!

Reply:
You only use/need 3 data sources. (user, shared, cached).
free (background) = 100 - (user + shared + cached)